plugins/modules/cassandra_keyspace.py

```diff
@@ -250,10 +261,17 @@ def main():
                " install cassandra-driver.")
         module.fail_json(msg=msg)
 
+    if HAS_SSL_LIBRARY is False:
```

Comment:
Let's also check if the ssl parameter is true here as well. That way we only require the ssl lib when it's actually used.

Reply:
Yep, done